lib/src/current_weather/presentation/pages/current_weather_page.dart

```diff
@@ -22,6 +24,23 @@ class CurrentWeatherPage extends HookConsumerWidget {
     );
     final currentWeather = ref.watch(currentWeatherProvider);
 
+    _showSnackBar(BuildContext context, String message, Color? color) {
```

Comment:
This is a closure inside `build`, you don't need to re-define and pass `context`. If anything, since you use this closure once, just show your SnackBar directly where it's needed.